src/DurableTask.SqlServer/SqlTypes/MessageIdSqlType.cs

```diff
@@ -23,21 +21,23 @@ static class MessageIdSqlType
         public static SqlParameter AddMessageIdParameter(
             this SqlParameterCollection commandParameters,
             string paramName,
-            IEnumerable<TaskMessage> messageCollection)
+            IEnumerable<TaskMessage> messageCollection,
+            string schemaName)
         {
             SqlParameter param = commandParameters.Add(paramName, SqlDbType.Structured);
-            param.TypeName = SqlTypeName;
+            param.TypeName = $"{schemaName}.MessageIDs";
```

Comment:
Looks like we are using a literal here instead of SqlTypeName

Reply:
Replaced the literal with the extension method from the last suggestion.